Libraries/Opc.Ua.Client/SessionConfiguration.cs

```diff
@@ -44,8 +45,32 @@ namespace Opc.Ua.Client
     [KnownType(typeof(X509IdentityToken))]
     [KnownType(typeof(IssuedIdentityToken))]
     [KnownType(typeof(UserIdentity))]
+#if ECC_SUPPORT
+    [KnownType(typeof(ECParameters))]
+#endif
     public class SessionConfiguration
     {
+#if ECC_SUPPORT
+        /// <summary>
+        /// Creates a session configuration
+        /// </summary>
+        internal SessionConfiguration(ISession session,
+            byte[] serverNonce,
+            string userIdentityTokenPolicy,
+            Nonce eccServerEphemeralKey,
+            NodeId authenthicationToken)
+        {
+            SessionName = session.SessionName;
+            SessionId = session.SessionId;
+            AuthenticationToken = authenthicationToken;
+            Identity = session.Identity;
+            ConfiguredEndpoint = session.ConfiguredEndpoint;
+            CheckDomain = session.CheckDomain;
+            ServerNonce = serverNonce;
```

Comment:
are serverNonce and eph key are mutually exclusive? IS eccServerEphemeralKey also a byte[]?